tools/lit-stories/.storybook-shared/preview.css

```diff
@@ -0,0 +1,5 @@
+@chui lit-storybook;
+
+:root{
+    font-family: sans-serif;
```

Comment:
2 chars

Reply:
How do you mean?

Reply:
I forgot about `import '@dxos-theme'`, will do that instead.